draft-ietf-openpgp-pqc.md

```diff
@@ -724,7 +726,7 @@ of Curve448.
 The operation `x448.Encaps()` is defined as follows:
 
  1. Generate an ephemeral key pair {`v`, `V`} via `V = X448(v,U(P))` where `v`
-    is a random scalar
+    is a randomly generated octet string with a length 56 octets
```

Comment:
for X25519, it says "with a length *of* 32 octets".  This omits the "of".  maybe align them?

Reply:
Yes, clearly the "of" was missing. Added it now.